vowpalwabbit/cbify.cc

```diff
@@ -88,15 +88,19 @@ void finish(cbify& data)
 {
   CB::cb_label.delete_label(&data.cb_label);
   data.a_s.delete_v();
+
   if (data.use_adf)
   {
     for (size_t a = 0; a < data.adf_data.num_actions; ++a)
-    {
-      data.adf_data.ecs[a]->pred.a_s.delete_v();
-      VW::dealloc_example(CB::cb_label.delete_label, *data.adf_data.ecs[a]);
-      free_it(data.adf_data.ecs[a]);
-    }
+      {
```

Comment:
[Minor][Nit] Is the extra indentation intended?

Reply:
No.  

The problem with styles, in my experience, is that every editor has a different interpretation of precisely what a style is...